packages/link/package.json

```diff
@@ -24,7 +24,11 @@
     "jest-environment-jsdom": "^29.7.0"
   },
   "dependencies": {
-    "@meshconnect/node-api": "^2.0.0"
+    "@meshconnect/node-api": "^2.0.0",
+    "@tanstack/react-query": "^5.17.9",
```

Comment:
please remove this package

Reply:
`@tanstack/react-query` is already removed
and the latest `@meshconnect/node-api` is added